specification_documents/examples/intro_set.mzQC

```diff
@@ -308,15 +329,10 @@
       }
     ],
     "controlledVocabularies": [
-      {
-        "name": "Proteomics Standards Initiative Quality Control Ontology",
-        "uri": "https://github.com/HUPO-PSI/mzQC/blob/main/cv/qc-cv.obo",
-        "version": "1.0.0"
-      },
       {
         "name": "Proteomics Standards Initiative Mass Spectrometry Ontology",
-        "uri": "https://github.com/HUPO-PSI/psi-ms-CV/releases/download/v4.1.71/psi-ms.obo",
-        "version": "4.1.71"
+        "uri": "https://github.com/HUPO-PSI/psi-ms-CV/releases/download/v4.1.130/psi-ms.obo",
```

Comment:
needs whatever was used in https://github.com/HUPO-PSI/psi-ms-CV/pull/246 or later